<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>export_logger</class>
 <widget class="QDialog" name="export_logger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geolimi - Export Logger - Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_chemin">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(209, 223, 245);</string>
       </property>
       <property name="text">
        <string>chemin du fichier</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_recherche">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Choix du chemin où enregistrer le fichier exporté</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="cb_liste">
     <property name="toolTip">
      <string>Sélection du district de trie pour l'export</string>
     </property>
     <item>
      <property name="text">
       <string>Liste des districts</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pb_logger">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_quitter">
       <property name="toolTip">
        <string>Sortir de la fenêtre</string>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_export">
       <property name="toolTip">
        <string>Exporter les informations de la base dans un fichier</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
